files/build_templates/swss.service.j2

```diff
@@ -40,6 +41,7 @@ ExecStopPost=/usr/bin/mst stop
 {% elif sonic_asic_platform == 'cavium' %}
 ExecStopPost=/etc/init.d/xpnet.sh stop
 ExecStopPost=/etc/init.d/xpnet.sh start
+ExecStopPost=/usr/bin/accton_as7512_util.py clean
```

Comment:
in this case, can it be moved to the the as7512 platform package instead of here?